pkg/rancher-desktop/components/MarketplaceCatalog.vue

```diff
@@ -0,0 +1,49 @@
+<script lang="ts">
+import Vue from 'vue';
+
+import { demoMarketplace } from '../utils/_demo_marketplace_items.js';
+
+import MarketplaceCard from '@pkg/components/MarketplaceCard.vue';
+
+type FilteredExtensions = typeof demoMarketplace.summaries;
+
+export default Vue.extend({
+  name:       'marketplace-catalog',
+  components: { MarketplaceCard },
+  data() {
+    return {
+      searchValue: '',
+      extensions:  demoMarketplace.summaries.slice(0, 2),
```

Comment:
What's the reason for the `slice` ? This needs a comment if it's intended that way

Reply:
This is a holdover from the original implementation that contained more summaries, some of which weren't relevant. 

This component is transferred over from the original `Marketplace.vue` implementation to accommodate the inclusion of tabs, so I'd rather not change things too much in this PR. I can definitely add a comment.

Reply:
After thinking about this for a moment, I opted to just list all of the demo summaries. We can slice the data all we want if there's a need for it in the future.